.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 before_install:
   - npm install -g npm@latest
 install:
-  - npm install --global electron
+  - npm install --global electron@1.4.4
```

Comment:
I'm probably missing something obvious, but what's wrong with making electron a dev-dependency in `package.json` as @jviotti suggested?